app/models/capital.rb

```diff
@@ -27,6 +23,54 @@ def complete?
     valid?(:submission)
   end
 
+  def savings
+    return [] unless requires_full_capital?
+
+    crime_application.savings.where(ownership_type: ownership_types)
+  end
+
+  def properties
+    return [] unless requires_full_capital?
+
+    crime_application.properties
```

Comment:
does this require `.where(ownership_type: ownership_types)`

Reply:
We could investigate the percentage owned values to determine if we should include properties not owned by the applicant. For now, we rely on the applicant to remove any entries added when a partner was included in the means.